api/networkserver.proto

```diff
@@ -47,7 +47,10 @@ service AsNs {
 
 // The GsNs service connects a Gateway Server to a Network Server.
 service GsNs {
+  // HandleUplink forwards an uplink message to the Network Server.
```

Comment:
No need to follow godoc here (`<symbol> <verb>`). In fact, it looks a bit weird in the generated docs.